fields/class-cmb-number-field.php

```diff
@@ -30,9 +32,13 @@ public function default_args() {
 	 * Print out field HTML.
 	 */
 	public function html() {
+		$attrs = [];
+		$attrs[] = '' !== $this->args['step'] ? sprintf( 'step="%g"', $this->args['step'] ) : '';
+		$attrs[] = '' !== $this->args['min'] ? sprintf( 'min="%g"', $this->args['min'] ) : '';
+		$attrs[] = '' !== $this->args['max'] ? sprintf( 'max="%g"', $this->args['max'] ) : '';
```

Comment:
`%g` is meant to only allow double values, not sure if we need `esc_attr` with that